eskip/eskip.go

```diff
@@ -96,6 +96,21 @@ type Route struct {
 	Backend string
 }
 
+type RoutePredicate func(*Route) bool
```

Comment:
this is a trivial type and not used by this package. Also, since I am not sure yet, if the DeleteAllIf methods in the other packages are really need to be public and in those packages, I think this is not required here.